app/models/block.rb

```diff
@@ -56,4 +63,10 @@ def mark_messages_read
       message.save
     end
   end
+
+  def invalidate_caches
+    return unless created_at == updated_at || saved_change_to_hide_me? || saved_change_to_hide_them?
```

Comment:
can we not use something like new_record? here, I worry about the created/updated check. for that matter, do new records automatically trigger the saved_change_to thing?

Reply:
We can't use new_record? because it's... already saved... because we're using after_commit. I think this is the closest I found. And I don't think so but I can check I suppose? I remember getting this working was an Adventure.